app/west.yml

```diff
@@ -35,12 +35,21 @@ manifest:
     - name: lvgl
       path: modules/lib/gui/lvgl
       revision: 74fc2e753a997bd71cefa34dd9c56dcb954b42e2
+    - name: littlefs
+      path: modules/fs/littlefs
+      revision: 0aefdda69d236cb01f932117c1f32e9da09c0ec3
     - name: mcuboot
       revision: e88113bbebe34ff2ccc6627ffae885cfeed6fdfd
       path: bootloader/mcuboot
+    - name: mcumgr
+      revision: 5885efb7cabf7b566577b73129c9d277d7d8848d
+      path: modules/lib/mcumgr
     - name: segger
       path: modules/debug/segger
       revision: 6fcf61606d6012d2c44129edc033f59331e268bc
+    - name: tinycbor
```

Comment:
why do. you need this library for ?

Reply:
It's required by mcumgr. Excluding it leads to:
```
pinetime-hypnos/modules/lib/mcumgr/mgmt/include/mgmt/mgmt.h:24:10: fatal error: tinycbor/cbor.h: No such file or directory
   24 | #include "tinycbor/cbor.h"
```